<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <title>Arun Boddapati - Resume</title>
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <div class="dark-mode-toggle">
            <label class="switch">
                <input type="checkbox" id="darkModeToggle" onclick="toggleDarkMode()">
                <span class="slider round"></span>
            </label>
        </div>
    <h1>Arun Boddapati</h1>
    <p>Bioinformatics Scientist | Machine Learning Expert | Pipeline Developer</p>
    <p><a href="mailto:arunbodd@icloud.com">arunbodd@icloud.com</a> | <a href="https://www.linkedin.com/in/arunbodd/">LinkedIn</a> | Atlanta, Georgia</p>

    <!-- Tab links -->
    <div class="tab">
        <button class="tablinks" onclick="openTab(event, 'Resume')">Resume</button>
        <button class="tablinks" onclick="openTab(event, 'Publications')">Publications</button>
    </div>

    <!-- Tab content -->
    <div id="Resume" class="tabcontent">
        <h2>Resume</h2>
        <button class="collapsible">About Me</button>
        <div class="content">
            <p>As a versatile full-stack bioinformatics scientist, I thrive in collaborative environments, having worked seamlessly across diverse teams ranging from Genomics Core to Scientific Computing. My commitment to meeting client expectations has been a cornerstone of my success, as evidenced by my track record of delivering high-quality results in alignment with client needs. With expertise spanning NGS data analysis, pipeline development, machine learning, and tool development, I bring a holistic approach to problem-solving. Through effective project management, team building, and mentoring, I drive initiatives that not only meet but exceed customer expectations, while advancing the forefront of data science and biomedical research.</p>
        </div>

        <button class="collapsible">Education</button>
        <div class="content">
            <h3>M.S. in Bioinformatics</h3>
            <p>Indiana University-Purdue University (IUPUI), 2016 - 2018, Indianapolis, Indiana</p>
            <h3>M.S in Biomedical Science</h3>
            <p>Symbiosis School of Biological Sciences, 2012 - 2014, Pune, India</p>
            <h3>Bachelors in Biotechnology</h3>
            <p>Sreenidhi Institute of Science and Technology, 09/2007 - 07/2011, Hyderabad, India</p>
        </div>

        <button class="collapsible">Experience</button>
        <div class="content">
            <h3>Bioinformatics Scientist</h3>
            <p>Leidos, 06/2022 - Present, Atlanta, Georgia</p>
            <ul>
                <li>Developed the National Wastewater Surveillance (NWSS) pipeline (Aquascope), significantly reducing analysis time by 80%, thereby improving the efficiency of wastewater monitoring efforts.</li>
                <li>Mentored a junior bioinformatician at CDC, providing guidance and support on various collaborative projects to ensure their professional development and project success </li>
                <li>Led a project investigating the impact of gene expression in patients exposed to Silica dust, utilizing machine learning and generative learning models to uncover insights. Also, mentored a summer intern and presented a paper at AMD Day'23</li>
                <li>Served as the training lead for Nextflow, Singularity, High-Performance Computing, and Transcriptomics data analysis, equipping team members with essential skills to enhance project efficiency and effectiveness</li>
            </ul>
            <h3>Sr. Bioinformatics Analyst</h3>
            <p>Emory Primate Research Center, 09/2020 - 06/2022, Atlanta, Georgia</p>
            <ul>
                <li>Led the analysis for the Wastewater assessment of COVID-19 samples collected from wastewater plants in Atlanta, GA, contributing to vital insights into community transmission patterns.</li>
                <li>As lead analyst, developed analysis pipelines for the IMPACC study at Emory, overseeing analysis of data from 5000 COVID-19 patients, resulting in three impactful publications in leading journals.</li>
                <li>Spearheaded the development of RNA-Seq and Single-Cell RNA-Seq pipelines, leading to the publication of four high-impact articles, while also providing training in single-cell RNA data analysis and bioinformatics tools to graduate students and staff scientists.</li>
            </ul>
            <h3>Bioinformatics Analyst II</h3>
            <p>Leidos Biomedical Research, 04/2018 - 09/2020, Bethesda, Maryland</p>
            <ul>
                <li>Performed comprehensive preliminary and exploratory data analysis for gene expression studies across various NIAID projects, leveraging Microarray, RNA-Seq, and Single Cell RNA sequencing data to uncover new leads and insights.</li>
                <li>Developed interactive visualizations using R markdown reports, facilitating easy understanding of results and contributing to informed decision-making processes.</li>
                <li>Streamlined bioinformatics workflows by developing pipelines for next-generation sequencing data analysis, resulting in significant time and cost savings, including quicker project turnaround times and reduced resource utilization.</li>
            </ul>
        </div>

        <button class="collapsible">Skills</button>
        <div class="content">
            <p><Strong>Management Skills</strong>: Project management, Team building, Quality management, Mentorship, Scrum, Jira</p>
            <p><Strong>Bioinformatics & Biomedical Sciences</Strong>: NGS Analysis, Infectious diseases, Immunology, Repertoire sequencing, Epidemiology, Nextflow, Snakemake, CI/CD Github, Single-cell, Cite-seq</p>
            <p><Strong>Programming/Scripting:</Strong> R/Rshiny/Quarto, Python/PyTorch, Shell, HTML, CSS</p>
            <p><Strong>High performance & Cloud Computing</Strong>: Singularity, Docker, AWS, Azure</p>
            <p><Strong>Machine Learning</Strong>: TensorFlow, Regression, Classification, Generative neural nets, NLP</p>
        </div>

        <button class="collapsible">Strengths</button>
        <div class="content">
            <p>Curiosity, Initiative, Problem solving, Project management, Team collaboration, Adaptibility, Time & Resource management, Leadership, Mentoring, Client relations, Conflict resolution, Strategic planning, Project planning</p>
        </div>

        <button class="collapsible">Passions</button>
        <div class="content">
            <p>Tech enthusiast, Hiking aficionado, Soccer Buff</p>
        </div>

        <button class="collapsible">Find Me Online</button>
        <div class="content">
            <p>Google Scholar: <a href="https://scholar.google.com/citations?user=ni4A6KgAAAAJ&hl=en">Arun K. Boddapati</a></p>
            <p>GitHub: <a href="https://github.com/arunbodd">arunbodd</a></p>
            <p>LinkedIn: <a href="https://www.linkedin.com/in/arunbodd/">arunbodd</a></p>
        </div>
    </div>

    <div id="Publications" class="tabcontent">
        <h2>Publications</h2>
        <div class="publication-container"> <!-- Added this div -->
            <div class="publication">
                <img src="images/Baricitinib_fig4.jpg" alt="Baricitinib treatment" onclick="showDetails('pub1')">
                <div class="details" id="pub1">
                    <h3><a href="https://doi.org/10.1016/j.cell.2020.11.007" target="_blank">Baricitinib treatment resolves lower-airway macrophage inflammation and neutrophil recruitment in SARS-CoV-2-infected rhesus macaques</a></h3>
                    <p><strong>Abstract:</strong> SARS-CoV-2-induced hypercytokinemia and inflammation are critically associated with COVID-19 severity. Baricitinib, a clinically approved JAK1/JAK2 inhibitor, is currently being investigated in COVID-19 clinical trials. Here, we investigated the immunologic and virologic efficacy of baricitinib in a rhesus macaque model of SARS-CoV-2 infection. Viral shedding measured from nasal and throat swabs, bronchoalveolar lavages, and tissues was not reduced with baricitinib. Type I interferon (IFN) antiviral responses and SARS-CoV-2-specific T cell responses remained similar between the two groups. Animals treated with baricitinib showed reduced inflammation, decreased lung infiltration of inflammatory cells, reduced NETosis activity, and more limited lung pathology. Importantly, baricitinib-treated animals had a rapid and remarkably potent suppression of lung macrophage production of cytokines and chemokines responsible for inflammation and neutrophil recruitment. These data support a beneficial role for, and elucidate the immunological mechanisms underlying, the use of baricitinib as a frontline treatment for inflammation induced by SARS-CoV-2 infection.</p>
                    <p><strong>Main Contributions:</strong></p>
                    <ul>
                        <li>Reduction in Inflammatory Cytokine Expression: Baricitinib treatment significantly reduces the expression of inflammatory cytokines (e.g., IL-6, TNF-a) in pulmonary macrophages.</li>
                        <li>Altered Chemokine Profiles: The treatment impacts the expression of chemokines involved in neutrophil and macrophage recruitment.</li>
                        <li>Visualization of Cellular Changes: UMAP projections and heatmaps provide a visual representation of the changes in cellular composition and gene expression due to baricitinib treatment.</li>
                        <li>Preservation of Interferon Responses: Despite the reduction in inflammatory chemokines, ISG expression remains largely unaffected by baricitinib.</li>
                        <li>Support for Therapeutic Impact: These detailed analyses support the conclusion that baricitinib reduces airway inflammation while maintaining antiviral immune responses.</li>
                    </ul>
                </div>
            </div>
            <div class="publication">
                <img src="images/MVA_S_vaccine.jpg" alt="MVA/S Vaccine" onclick="showDetails('pub2')">
                <div class="details" id="pub2">
                    <h3><a href="https://doi.org/10.1016/j.immuni.2021.02.001" target="_blank">A modified vaccinia Ankara vector-based vaccine protects macaques from SARS-CoV-2 infection, immune pathology, and dysfunction in the lungs</a></h3>
                    <p><strong>Abstract:</strong> Modified vaccinia Ankara (MVA) vector-based vaccines are attractive because of their excellent safety and ability to induce long-lived humoral and cellular immunity in humans. Routhu et al. show that an MVA-based COVID-19 vaccine encoding prefusion-stabilized spike (MVA/S) induces strong neutralizing antibody and CD8+ T cell responses and protects macaques from SARS-CoV2 infection, immunopathology, and infection-induced B cell abnormalities in the lungs.</p>
                    <p><strong>Main Contributions:</strong></p>
                    <ul>
                        <li><strong>Development of MVA-Based Vaccine:</strong> The study developed an MVA-based vaccine encoding a prefusion-stabilized spike protein (MVA/S) and demonstrated its efficacy in inducing strong neutralizing antibody and CD8+ T cell responses.</li>
                        <li><strong>Cell Composition:</strong> The majority of cells in the BAL fluid were CD68+ myeloid cells (82% in controls and 71% in vaccinated animals), followed by CD8+ T cells (7% in controls and 14% in vaccinated animals), with minor populations of B cells, dendritic cells (DCs), and epithelial cells.</li>
                        <li><strong>Single-Cell RNA Sequencing:</strong> The use of single-cell RNA sequencing to analyze the cellular response in the BAL fluid provided detailed insights into the immune mechanisms involved and the reduction of inflammatory responses due to vaccination.</li>
                        <li><strong>Durable Immune Response:</strong> The vaccine induced strong neutralizing antibodies and CD8+ T cell responses that were maintained over time, suggesting long-term protection potential.</li>
                        <li><strong>Gene Expression:</strong> In control animals, there was widespread induction of IFN-stimulated genes (ISGs) and inflammatory cytokines (e.g., IL-6, TNF-α, IL-1β) as well as chemokines (e.g., CXCL8, CXCL10). In vaccinated animals, the expression of these genes was significantly reduced.</li>
                        <li><strong>Potential for Human Use:</strong> The study supports the potential of the MVA/S vaccine as a candidate for human use, with the ability to induce durable immune responses and protection against SARS-CoV-2.</li>
                    </ul>
                </div>
            </div>
            <div class="publication">
                <img src="images/AtypicalB_cells.jpeg" alt="Shared Transcriptional Profiles" onclick="showDetails('pub3')">
                <div class="details" id="pub3">
                    <h3><a href="https://www.science.org/doi/10.1126/sciadv.abg8384" target="_blank">Shared transcriptional profiles of atypical B cells suggest common drivers of expansion and function in malaria, HIV, and autoimmunity</a></h3>
                    <p><strong>Abstract:</strong> Chronic infectious diseases have a substantial impact on the human B cell compartment including a notable
                        expansion of B cells here termed atypical B cells (ABCs). Using unbiased single-cell RNA sequencing (scRNA-seq),
                        we uncovered and characterized heterogeneities in naïve B cell, classical memory B cells, and ABC subsets. We
                        showed remarkably similar transcriptional profiles for ABC clusters in malaria, HIV, and autoimmune diseases and
                        demonstrated that interferon- drove the expansion of ABCs in malaria. These observations suggest that ABCs
                        represent a separate B cell lineage with a common inducer that further diversifies and acquires disease-specific
                        characteristics and functions. In malaria, we identified ABC subsets based on isotype expression that differed in
                        expansion in African children and in B cell receptor repertoire characteristics. Of particular interest, IgD+IgMlo and
                        IgD−IgG+ ABCs acquired a high antigen affinity threshold for activation, suggesting that ABCs may limit autoimmune
                        responses to low-affinity self-antigens in chronic malaria</p>
                    <p><strong>Main Contributions:</strong></p>
                    <ul>
                        <li><strong>Gene Expression Distribution (Panel A):</strong> ABCs display a unique transcriptional signature with 176 differentially expressed genes (DEGs), including a variety of functional proteins such as cell surface markers, cytokines, transcription factors, adhesion molecules, cytoskeleton regulators, signaling molecules, and metabolism-related proteins.</li>
                        <li><strong>Heatmap of DEGs (Panel B):</strong> The DEGs in ABCs compared to naïve B cells, cMBCs, and acMBCs show that cell surface markers such as FCRL5, LILRB2 (CD85D), ITGAX (CD11c), and CD72 are highly expressed in ABCs. Additionally, TFs like TBX21 (Tbet) and new ones like ZBTB32, NR4A2, TOX2, and TCF7 are differentially expressed in ABCs, along with cytokines and signaling molecules being part of the unique signature of ABCs.</li>
                        <li><strong>Validation of DEGs (Panel C):</strong> Flow cytometry confirms the expression of several DEGs identified in ABCs including CD72, CD79A, integrin β-2 (ITGB2), CD55 (down-regulated in ABCs), CD63, and CD84. High expression of CD79A suggests a distinctive B cell receptor (BCR) complex structure in ABCs.</li>
                        <li><strong>IL-10 Expression (Panel D):</strong> ABCs express higher levels of IL-10 mRNA and surface IL-10 receptor (IL10R) compared to naïve B cells and cMBCs, suggesting a role for ABCs in controlling inflammatory responses during febrile malaria.</li>
                        <li><strong>Transcription Factors (Panel E):</strong> Flow cytometry validates the up-regulation of ZBTB32, NR4A2, and TOX2, and down-regulation of TCF7 in ABCs. These TFs are associated with T cell exhaustion features, although ABCs are not functionally exhausted.</li>
                    </ul>
                </div>
            </div>
            <div class="publication">
                <img src="images/TIGIT_CD8.jpg" alt="MVA/S Vaccine" onclick="showDetails('pub4')">
                <div class="details" id="pub4">
                    <h3><a href="https://academic.oup.com/jid/article/224/9/1599/6179305" target="_blank">Correlation Between TIGIT Expression on CD8+ T Cells and Higher Cytotoxic Capacity</a></h3>
                    <p><strong>Abstract:</strong> Persistent exposure to antigen leads to T-cell exhaustion and immunologic dysfunction. We examined the immune exhaustion markers T cell immunoglobulin and ITIM domain (TIGIT) and programmed cell death protein 1 (PD-1) in human immunodeficiency virus (HIV)–infected and healthy individuals and the relationship with cytotoxic CD8+ T-lymphocyte activity. Frequencies of TIGIT but not PD-1 were positively correlated with CD8+ T-lymphocyte activity in HIV-aviremic and healthy individuals; however, there was no correlation in HIV-viremic individuals. Transcriptome analyses revealed up-regulation of genes associated with antiviral immunity in TIGIT+CD8+ versus TIGIT−CD8+ T cells. Our data suggest that TIGIT+CD8+ T cells do not necessarily represent a state of immune exhaustion and maintain an intrinsic cytotoxicity in HIV-infected individuals.</p>
                    <p><strong>Main Contributions:</strong></p>
                    <ul>
                        <li><strong>Distinct Transcriptional Profiles:</strong> The study identified distinct transcriptional profiles between TIGIT+ and TIGIT− CD8+ T cells, highlighting the difference in gene expression patterns which are more similar between HIV-aviremic and healthy individuals than with HIV-viremic individuals.</li>
                        <li><strong>Gene Ontology Analysis:</strong> The study revealed that genes associated with defense response, innate immune response, cell activation, positive regulation of cytokine production, and immune effector processes were enriched in TIGIT+CD8+ T cells.</li>
                        <li><strong>Differential Gene Expression:</strong> The study demonstrated that certain cytotoxic molecules and transcription factors (such as granzyme A, H, K, and transcription factors Eomesodermin and TOX) were up-regulated in TIGIT+CD8+ T cells, indicating an intrinsic capacity for cytotoxicity despite the expression of inhibitory receptors.</li>
                        <li><strong>Inhibitory Receptor Expression:</strong> The study found that the expression of multiple inhibitory receptors (including PD-1, CTLA-4, CD160, and CD244) was up-regulated in TIGIT+CD8+ T cells, particularly in HIV-viremic individuals, suggesting that these cells might be contributing to the impairment of CTL functions during active viral replication.</li>
                    </ul>
                </div>
            </div>             
        </div>
    </div>    

    <script src="script.js"></script>
</body>
</html>
